src/main/java/com/google/sps/data/Email.java

```diff
@@ -14,18 +14,55 @@
 
 package com.google.sps.data;
 
+import com.google.sps.servlets.Post;
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+
+/** Creates and converts HTML templates for email sending. */
 public final class Email {
 
   public static final String welcomeSubject = "⭐ noms: welcome to your free food finder!";
-  public static final String dailyDigestSubject = "⭐ noms: daily digest of free food!";
+  public static final String newPostSubject = "⭐ noms: new free food near you!";
+
+  // TODO: Find relative path instead of absolute (FileNotFoundException if just ./WelcomeEmail.html).
+  public static final String welcomeContentPath = "/home/areeta/noms/src/main/java/com/google/sps/data/WelcomeEmail.html";
+  public static final String newPostPath = "/home/areeta/noms/src/main/java/com/google/sps/data/NewPost.html";
```

Comment:
won't these two lines break if deployed from anywhere but your machine? :0 I know you have it listed as a TODO but I would recommend seeing if this can be addressed before merging

Reply:
Yes, they would! For this part, a lot of issues came up because running and deploying our project is from the `target` which places files differently than our local instance so I put in the absolute path here. However, I moved around the files so it would be seen which fixed this!